src/charging-station/ui-server/ui-services/AbstractUIService.ts

```diff
@@ -59,7 +60,10 @@ export default abstract class AbstractUIService {
     return JSON.stringify([command, payload]);
   }
 
-  private handleListChargingStations(): JsonType {
-    return Array.from(this.uiServer.chargingStations);
+  private handleListChargingStations = (): JsonType =>
+    Array.from(this.uiServer.chargingStations, ([_key, value]) => value as unknown as JsonType);
```

Comment:
Why an underscore?

Reply:
Because since `key` is not used, SonarCloud tells me to add an underscore. Not mandatory but doesn't cost much.